content/configuration/source.md

```diff
@@ -13,9 +13,68 @@ The source settings contains:
 
 Sources are bound to specific channels in the communications configuration. To learn more, read the [Source and Executor Bindings](/configuration/communication/#source-and-executor-bindings) section.
 
+## General merging strategy
+
+A source essentially notifies a channel about events for configured resources filtered by specified namespaces.
+
+When a channel binds to more than one source, the notifications are merged across all sources.
+
+Consider this example,
+
+### Example
+
+Let's say you have a resource defined in more than one source but wired with different events and namespaces.
+
+```yaml
+sources:
+  'k8s-events':
+    kubernetes:
+      resources:
+        - name: v1/configmaps
+          namespaces:
+            include:
+              - (botk.*|default)
+          events:
+            - create
+            - update
+            - delete
+  'k8s-updates':
+    kubernetes:
+      resources:
+        - name: v1/configmaps
+          namespaces:
+            include:
+              - botkube
+          events:
+            - update
+```
+
+The bound channel `monitor-config` (below) will notify on the merged events and namespaces across all resource/source definitions.
+
+```yaml
+communications:
+  'default-group':
+    slack:
+      # ... trimmed ...
+      channels:
+        'monitor-config':
+          name: "monitor-config"
+          # ... trimmed ...
+          bindings:
+            # ... trimmed ...
+            sources:
+              - k8s-events
+              - k8s-updates
+...
+```
+
+Meaning, channel `monitor-config` will notify on `v1/configmaps` events matching,
```

Comment:
channel is not a notifier. On a given channel, we will receive such notifications